scripts/estate.py

```diff
@@ -17,26 +17,6 @@ def deploy_estate_implementation(account: Optional[Account] = None) -> Contract:
     return estate
 
 
-def initialize_estate(
```

Comment:
Is this removed because the estate contract doesn't need to be initialized any more?

Reply:
Correct. The implementation contract is now initialized with the constructor.